ui/src/app/pages/workspace/workspace-edit.spec.tsx

```diff
@@ -1100,48 +1031,26 @@ describe('WorkspaceEdit', () => {
       },
     });
 
-    const wrapper = component();
-    await waitOneTickAndUpdate(wrapper);
-
-    const billingDropDown = wrapper
-      .find('[data-test-id="billing-dropdown"]')
-      .first();
+    renderComponent();
 
-    expect(billingDropDown.props().value).toEqual('free-tier');
-    // @ts-ignore
-    expect(billingDropDown.props().options.map((o) => o.value)).toEqual([
-      'free-tier',
-      'user-billing',
-    ]);
-    // @ts-ignore
-    expect(billingDropDown.props().options.map((o) => o.label)).toEqual([
-      'Use All of Us initial credits - $34.56 left',
-      'User Billing',
-    ]);
-  });
+    expect(await screen.findByDisplayValue(freeTierRegex)).toBeInTheDocument();
 
-  it('should show free tier and user billing account when they grant billing scope when creating workspace', async () => {
```

Comment:
Deleted, because it seems identical to 'should show free tier user account and user-billing when user granted billing scope'

Reply:
yeah I'm not seeing the difference